### DISCUSSION TITLE
Incorporation of newer reservoirs (post 2006)

### DISCUSSION THREAD

**Discussion by abbylute** [iiasa/CWatM#144] [Q&A]
Is there a plan to update the lake and reservoir input layers for CWatM? I noticed the last year in the 30 arcmin layer is 2006 and the last year in the 5 arcmin layer is 2008, so many newer reservoirs are missing. From this comment (https://github.com/iiasa/CWatM/discussions/114) it sounds like these layers are based on HydroLakes. In that case, is it a matter of waiting for HydroLakes to be updated? In the meantime, do you have any suggestions for adding newer reservoirs in the model?  Perhaps an interim solution could be to update the layers using GeoDAR (https://essd.copernicus.org/articles/14/1869/2022/)? Thanks!

**Comment by mikhailsmilovic** [comment]
This discussion goes through updating the lakes and reservoirs maps: https://github.com/iiasa/CWatM/discussions/169

Generally, updating the global lakes and reservoirs maps is a great idea.
If someone were to create these maps and test them, following the format of the previous ones, these could replace the current standard maps.
